pilot/lib/pilot/tester_manager.rb

```diff
@@ -55,8 +56,10 @@ def find_tester(options)
     def remove_tester(options)
       start(options)
 
-      tester = Spaceship::Tunes::Tester::External.find(config[:email])
-      tester ||= Spaceship::Tunes::Tester::Internal.find(config[:email])
+      tester = Spaceship::Tunes::Tester::Internal.find(config[:email])
+      UI.user_error!("#{tester.email} is an internal tester; pilot does not support internal testers") unless tester.nil?
+
+      tester = Spaceship::Tunes::Tester::Internal.find(config[:email])
```

Comment:
Since the primary use-case is to find external testers, lets re-order this logic like this:
* find the external tester, and move on.
* if it the external tester can't be found, check to see if the supplied email is an internal user
* if the internal user is found, throw the exception saying we found it, but it's internal and we dont support that
* if no user is found, throw an exception saying that user can't be found.

thoughts?